code/modules/power/apc.dm

```diff
@@ -581,6 +581,32 @@
 			return TRUE
 
 /obj/machinery/power/apc/attackby(obj/item/W, mob/living/user, params)
```

Comment:
Change obj/item/W.
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
I know you didn't add it, but it might be good to change it since you're adding more instances of it

Reply:
> Exceptions can be made in the case of inheriting existing procs, as it makes it so you can use named parameters, but new variable names must follow these standards. It is also welcome, and encouraged, to refactor existing procs to use clearer variable names.

Reply:
Though it's good to just immediately `var/obj/item/item = W` if you *really* wanted to, but the proper solution is fixing `/attackby` as a whole--BYOND doesn't play well when the same proc has different signatures.